### PR TITLE
Add known app issue in 2.5

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/25445

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -857,6 +857,12 @@ The _Console_ and _Topology_ for _Application_ changes for the {product-version}
 
 When you create `ApplicationSet` applications that deploy resources to a different namespace than the namespace defined in the `ApplicationSet` YAML, the resource status does not appear in the topology.
 
+[#allow-deny-list-not-working-objectstorage-app]
+=== Allow and deny list does not work in Object storage applications
+// 2.6:25445
+
+The `allow` and `deny` list feature does not work in Object storage application subscriptions.
```
Comment (dockerymick) [inline release_notes/known_issues.adoc:864]: Does "Object" need to be capitalized? I recommend using lowercase "o". unless you want to highlight that this is the name of the subscription? Maybe consider italicizing?

**Review by dockerymick — APPROVED** [review]
Left a comment

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (oafischer) [reply release_notes/known_issues.adoc:864]: Good question, I've seen both in the doc. About 70% of instances is capitalized, the other 30% is not. I'd say capitalize based on that but I'll ask Chris.

Comment (oafischer) [reply release_notes/known_issues.adoc:864]: He says in this case it would be capitalized.

Comment (dockerymick) [reply release_notes/known_issues.adoc:864]: Sounds good!

**Review by rokej — APPROVED** [review]
Thank you!

**Pull request merged**